backend/server.js

```diff
@@ -34,11 +39,8 @@ app.get("/translation", async(req, res) => {
   const translator = new deepl.Translator(authKey);
 
   try {
-    (async () => {
-      const result = await translator.translateText(textToTranslate, inputLanguage, outputLanguage);
-      console.log(result.text);
-      res.status(200).json(result.text);
-    })();
+    const result = await translator.translateText(textToTranslate, inputLanguage, outputLanguage);
+    res.status(200).json(result);
```

Comment:
You do not need to use `async` again here before you can use `await`, the function is already wrapped as `async` here i.e `app.get("/languages", **async**(req, res) => {`

Reply:
I got it.